### ISSUE TITLE
TypeError: unsupported operand type(s) for -: 'int' and 'str'

### ISSUE THREAD

**Issue by akrisha20** [mctinker/MapTasker#15]
Got the following error in the maptasker_debug.log, when performing the command "maptasker -o"

```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Users\robin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\Scripts\maptasker.exe\__main__.py", line 7, in <module>
  File "C:\Users\robin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\maptasker\main.py", line 28, in main
    return_code = mapit.mapit_all("")
                  ^^^^^^^^^^^^^^^^^^^
  File "C:\Users\robin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\maptasker\src\mapit.py", line 698, in mapit_all
    special_handling(found_tasks, projects_without_profiles, projects_with_no_tasks)
  File "C:\Users\robin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\maptasker\src\mapit.py", line 640, in special_handling
    display_back_matter(
  File "C:\Users\robin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\maptasker\src\mapit.py", line 502, in display_back_matter
    process_outline()
  File "C:\Users\robin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\maptasker\src\mapit.py", line 418, in process_outline
    outline_the_configuration()
  File "C:\Users\robin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\maptasker\src\outline.py", line 519, in outline_the_configuration
    network_map(network)
  File "C:\Users\robin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\maptasker\src\diagram.py", line 812, in network_map
    build_network_map(network)
  File "C:\Users\robin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\maptasker\src\diagram.py", line 757, in build_network_map
    PrimeItems.netmap_output = handle_calls(PrimeItems.netmap_output)
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\robin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\maptasker\src\diagram.py", line 587, in handle_calls
    draw_arrows_to_called_task(up_down_location, value, output_lines)
  File "C:\Users\robin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\maptasker\src\diagram.py", line 474, in draw_arrows_to_called_task
    add_down_and_up_arrows(
  File "C:\Users\robin\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\LocalCache\local-packages\Python311\site-packages\maptasker\src\diagram.py", line 410, in add_down_and_up_arrows
    line_to_modify = called_line_num - called_line_index
                     ~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~
TypeError: unsupported operand type(s) for -: 'int' and 'str'

```

**Comment by mctinker** [comment]
Can you please send me your backup XML file to mikrubin@gmail.com?

**Comment by mctinker** [comment]
This is a non-trivial bug.  I will post here as soon as a fix is available.

**Comment by mctinker** [comment]
The problem is now resolved with the new release version 3.0.5: "pip install maptasker --upgrade" .  Thanks for your patience.
